projects/plugins/jetpack/modules/wordads/php/class-wordads-smart.php

```diff
@@ -310,11 +291,44 @@ private function get_language_keywords(): array {
 	}
 
 	/**
-	 * Gets the blog's ID.
+	 * Enable formats by post types and the display options.
+	 *
+	 * @return void
+	 */
+	private function enable_formats(): void {
+		$this->formats['top']['enabled']                  = $this->params->options['enable_header_ad'];
+		$this->formats['inline']['enabled']               = is_singular( 'post' ) && $this->params->options['wordads_inline_enabled'];
+		$this->formats['belowpost']['enabled']            = $this->params->options['wordads_second_belowpost'];
+		$this->formats['bottom_sticky']['enabled']        = $this->params->options['wordads_bottom_sticky_enabled'];
+		$this->formats['sidebar']['enabled']              = $this->params->options['wordads_sidebar_enabled'];
```

Comment:
Do you think we should omit everything sidebar for now? I think we decided to implement sidebar later (probably after the migration), considering the code to hook into the theme sidebar doesn't exist currently.